docs/06-concepts/11-authentication/04-providers/06-custom-providers.md

```diff
@@ -165,3 +248,35 @@ if (serverResponse.success) {
 ```
 
 The example above shows how to register a signed-in user in the session manager.
+
+### Sign Out
+
+The `SessionManager` provides multiple ways to handle sign-out functionality, allowing you to either sign out from the current device or all devices:
+
+1. **signOutDevice**:  
+   This method allows the user to sign out of the current device while remaining signed in on other devices.
```

Comment:
Suggestion: Rephrase to `This method allows the user to only sign out of the current device. This is the most common scenario when signing out.`

Reply:
@SandPod check the new doc updates and let me know if this is okay or not?!